src/ICSharpCode.SharpZipLib/Zip/ZipFile.cs

```diff
@@ -2038,15 +2038,17 @@ private void WriteLocalEntryHeader(ZipUpdate update)
 				}
 				else if (entry.CompressionMethod == CompressionMethod.Stored)
 				{
-					entry.Flags &= ~(int)GeneralBitFlags.Descriptor;
+					// Older versions of pkzip would not accept Descriptor flag for non-deflate compression
+					// but disregarding that allows us to use Stored method without seeking!
+					// entry.Flags &= ~GeneralBitFlags.Descriptor;
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/piksel/SharpZipLib/pullRequest?prid=5745485)